### PR TITLE
Fix encode/decode imports in __init__.py

### PR DESCRIPTION
**Pull request by will-moore**
This is needed to fix my omero-web unit tests with py3.

### CODE REVIEW

File: omero_marshal/__init__.py [hunk 1]
```diff
@@ -13,8 +13,8 @@
 import pkgutil
 import logging
 import re
-from encode import encoders
-from decode import decoders
+from omero_marshal.encode import encoders
```
Comment (sbesson) [inline omero_marshal/__init__.py]:
```suggestion
from .encode import encoders
```

File: omero_marshal/__init__.py [hunk 2]
```diff
@@ -13,8 +13,8 @@
...
+from omero_marshal.decode import decoders
```
Comment (sbesson) [inline omero_marshal/__init__.py]:
```suggestion
from .decode import decoders
```

**Comment by sbesson** [comment]
Merging as this is a straight-forward fix that should be both Python2/Python3 compatible. As discussed with @ome/python, we can decide either to release this as `0.6.2` immediately or wait until we get a few more fixes related to the Python 3 work.

**Comment by joshmoore** [comment]
Tagging per https://github.com/ome/omero-web/pull/30#issuecomment-546267222

**Pull request merged**